### DISCUSSION TITLE
Request for Customizing Vim Mode Indicator in Zed

### DISCUSSION THREAD

**Discussion by soomtong** [zed-industries/zed#14391] [Feature Requests]
Thank you for implementing the vim mode in zed. However, is there a way to customize the bottom indicator that displays the mode? The UI used in ideavim(jetbrains) and neovim are as follows. 

<img width="305" alt="image" src="https://github.com/user-attachments/assets/a93290fc-3eeb-4773-b94c-8cac78d4f852">

<img width="384" alt="image" src="https://github.com/user-attachments/assets/cea1fd1e-0496-4b87-ba52-27627e45bda8">

Is it possible to make similar changes in zed? At the very least, removing the `--` would make it look cleaner.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
